dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -10466,23 +10617,6 @@ components:
             items:
               $ref: '#/components/schemas/WebhookRepository'
       description: A model for a GitHub webhook installation event
-      example: |
```

Comment:
Doesn't appear to have been moved elsewhere in openapi.yaml (not reviewing files in order top to bottom, so may be answered elsewhere), intentional?

Reply:
Yes intentional. The problem was that in the swagger UI, this rendered as a string, not a JSON object which made it difficult to work with due to escaped quotes. 

<img width="2836" height="730" alt="Image" src="https://github.com/user-attachments/assets/5b7dd5da-bb41-477d-ab45-40d478c32e0d" />

When the example is removed, it falls back on the `InstallationRepositoriesPayload` schema example which is easier to work with, but the downside is that it includes the `repository` field which isn't required for the install event. I tried to remove it (can probably see the InstallationRepositoriesPayload file where I overrode `getRepository()` to hide the property)  but seems like it didn't work